package.json

```diff
@@ -9,9 +9,17 @@
     "lint": "next lint"
   },
   "dependencies": {
+    "@fortawesome/fontawesome": "^1.1.8",
+    "@fortawesome/fontawesome-free-solid": "^5.0.13",
+    "@fortawesome/fontawesome-svg-core": "^6.1.1",
+    "@fortawesome/free-solid-svg-icons": "^6.1.1",
+    "@fortawesome/react-fontawesome": "^0.1.18",
+    "bootstrap": "^5.1.3",
```

Comment:
Can we use [Material UI v5.x](https://mui.com/material-ui/getting-started/installation/) instead? I don't use Bootstrap in my project. Material UI has [many nice components](https://mui.com/material-ui/react-button/) to use, just remember to use `Box` for `div`, and then use `sx` instead of `className`. You will love it! 🙂